examples/typescript/aws-s3-static-website/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "cdktf",
```

We require specific naming for examples as our CI tooling will then automatically pick them up and run them.
```suggestion
  "name": "@examples/typescript-aws-s3-static-website",
```